controller.go

```diff
@@ -547,6 +563,11 @@ func (c *controller) Sandboxes() []Sandbox {
 
 	list := make([]Sandbox, 0, len(c.sandboxes))
 	for _, s := range c.sandboxes {
+		// Hide stub sandboxes from libnetwork users
+		if s.isStub {
+			continue
+		}
```

Comment:
Wont this break the case of trying to clean the (ungracefully left-over) sandbox properly when the container is started (especially for the Global scoped networks and when the store is down during the daemon startup).
https://github.com/docker/docker/blob/master/daemon/container_unix.go#L898 in particular.